vantiq-cloud-infra-operations/terraform_azure/new/modules/aks/aks.tf

```diff
@@ -10,15 +10,11 @@ resource "azurerm_kubernetes_cluster" "aks-vantiq" {
   resource_group_name     = azurerm_resource_group.rg-aks.name
   dns_prefix              = var.aks_cluster_name
   private_cluster_enabled = var.private_cluster_enabled
-  role_based_access_control {
-    enabled = true
-  }
-  kubernetes_version  = var.kubernetes_version
-  node_resource_group = "${var.resource_group_name}-node"
+  kubernetes_version      = var.kubernetes_version
+  node_resource_group     = "${var.resource_group_name}-node"
 
   network_profile {
     dns_service_ip     = var.dns_service_ip
-    docker_bridge_cidr = var.docker_bridge_cidr
```

Comment:
Will be deprecated. `Optional`
- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#docker_bridge_cidr